recipes-bsp/uefi/tegra-uefi-capsules_35.3.1.bb

```diff
@@ -71,3 +71,21 @@ do_install() {
 
 FILES:${PN} += "${CAPSULE_DIR}"
 PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+do_deploy() {
+    install -d ${DEPLOYDIR}
+    if [ -e ${B}/tegra-bl.cap ]; then
+	BL_NAME=${TNSPEC_MACHINE}-tegra-bl.cap
+	install -m 0644 ${B}/tegra-bl.cap ${DEPLOYDIR}/$BL_NAME
+	ln -s -r ${DEPLOYDIR}/$BL_NAME ${DEPLOYDIR}/tegra-bl.cap
+    fi
+    if [ -e ${B}/tegra-kernel.cap ]; then
+	KERNEL_NAME=${TNSPEC_MACHINE}-tegra-kernel.cap
+	install -m 0644 ${B}/tegra-kernel.cap ${DEPLOYDIR}/$KERNEL_NAME
+	ln -s -r ${DEPLOYDIR}/$KERNEL_NAME ${DEPLOYDIR}/tegra-kernel.cap
+    fi
+}
+
+addtask deploy after do_install
+
+do_compile[depends] += "tegra-bup-payload:do_install"
```

Comment:
Good catch on the missing dependency, but the value here should really be `"${@bup_dependency(d)}"`, just like the `do_install` dependency in the `tegra-bup-payload` recipe.

Reply:
should be corrected now.